public/js/innerTextRemove.js

```diff
@@ -0,0 +1,3 @@
+const innerTextRemove = arrayOfElements => arrayOfElements.forEach((element) => {
+  element.value = '';
+});
```

Comment:
@aminalakhsham 👍
If it helps, you could combine this function, validation, and selectorFunction into a single file (for example ‘utils.js’, and then you could bring all of the functions into your views just by including that one file. Or if that one file gets too big or you like the organization better, you could do a ‘domUyils.js’ file and a ‘validation.js’ file. Whatever feels easiest to you!